cscs-checks/prgenv/mpi_t.py

```diff
@@ -48,6 +48,10 @@ def set_sanity(self):
                 'control': 'ref/mpit_control_vars_8.1.5.ref',
                 'categories': 'ref/mpit_categories_8.1.5.ref',
             },
+            '8.1.12': {
```

Comment:
Test cray-mpich/7.7.18 on dom :ok: 
```
[       OK ] (1/4) mpit_check @dom:gpu+PrgEnv-cray
[       OK ] (2/4) mpit_check @dom:gpu+PrgEnv-gnu
[       OK ] (3/4) mpit_check @dom:gpu+PrgEnv-intel
[       OK ] (4/4) mpit_check @dom:gpu+PrgEnv-nvidia
```
remove `PrgEnv-pgi` from the  list please

Reply:
Do you mean to remove `PrgEnv-pgi` from `self.valid_prog_environs` on line 19?